cornflow-server/cornflow/cli/migrations.py

```diff
@@ -41,6 +41,20 @@ def upgrade_migrations():
         upgrade()
 
 
+@migrations.command(name="downgrade", help="Downgrade migrations")
+def downgrade_migrations():
+    app = get_app()
+    external = int(os.getenv("EXTERNAL_APP", 0))
+    if external == 0:
+        path = "./cornflow/migrations"
+    else:
+        path = f"./{os.getenv('EXTERNAL_APP_MODULE', 'external_app')}/migrations"
+
+    with app.app_context():
+        migration_client = Migrate(app=app, db=db, directory=path)
+        downgrade()
```

Comment:
For the downgrade you should pass a migration file name or version to downgrade to that version. 
We could also do that for the upgrade as well.